core/src/space/hypeo/mankomania/StageFactory.java

```diff
@@ -392,4 +383,130 @@ public static void setFieldInfoImage(Texture texture){
 
     }
 
+    /**
+     * Shows the network-lobby:
+     * The lobby is a Scene in the game for players to join before playing the actual game.
+     * In the lobby, players can pick options and set themselves as ready for the game to start.
+     *
+     * The list in the lobby creates each player - both host and clients - itself.
+     * Therefore the hashmap 'players' - a field of class MHost - contains the necessary data.
+     * @param viewport
+     * @param stageManager
+     * @return stage/view of lobby
+     */
+    public static Stage getLobbyStage(final Viewport viewport, final StageManager stageManager)
```

Comment:
To keep StageFactory as small and clean as possible, please move this to a dedicated class LobbyStage (extends Stage). See also issue #50